docs-src/tutorials/02-usage.md

```diff
@@ -265,9 +268,9 @@ yourApp.on('some-event', () => {
 
 ### Rendering Tours in Specific Locations
 
-By default, tour steps will append their elements to the `body` element of the DOM. This is perfect for most use cases, but not always. If you need to have steps appended elsewhere you can take advantage of Tippy's
-[`appendTo` option](https://atomiks.github.io/tippyjs/#append-to-option) by defining it on the
-`tippyOptions` hash inside of each Step's options hash.
+By default, tour steps will append their elements to the `body` element of the DOM. This is perfect for most use cases, but not always. If you need to have steps appended elsewhere you can take advantage of Popper's
+[options](https://popper.js.org/docs/v2/constructors/#options) by defining it on the
```

Comment:
What popper option would be used for this? We may want to be explicit here.

Reply:
Actually, this is a tippy option and we easily do this with `Step.target`. Going to remove.

Reply:
I don't think `step.target` is the same thing. This is defining the container the step element itself lives in right?

Reply:
if you look at the docs on `createPopper()` The first arg is target and we map target there. Maybe I'm not clear on what you mean then?

Reply:
Target is what you are attaching the popper to. `appendTo` is where the tooltip element itself lives.